gateway/enforcer/build.gradle

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'base'
+    id 'jacoco-report-aggregation'
+}
+
+apply from: "$rootDir/../../common-gradle-scripts/docker.gradle"
+apply from: "$rootDir/../../common-gradle-scripts/java.gradle"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    jacocoAggregation project(':org.wso2.choreo.connect.enforcer.commons')
+    jacocoAggregation project(':org.wso2.choreo.connect.enforcer')
+}
+
+// TODO (amali) uncomment after fixing dependency issues
+// reporting {
+//     reports {
+//         testCodeCoverageReport(JacocoCoverageReport) {
+//             testType = TestSuiteType.UNIT_TEST
+//         }
+//     }
+// }
+
+// tasks.named('check') {
+//     dependsOn tasks.named('testCodeCoverageReport', JacocoReport)
+// }
+
+tasks.named('build').configure {
+    dependsOn docker_build
+}
+
+subprojects {
```

Comment:
This can be changed to `allprojects` and then we can remove apply from: lines (23 and 24) above.